features/naam/naam.feature

```diff
@@ -49,130 +47,227 @@
-      | datumTot                         | 2014-08-01         |
-      | adresseerbaarObjectIdentificatie | 0800000000000001   |
-      Dan heeft de response een bewoning met de volgende gegevens
-      | naam                             | waarde                    |
-      | periode                          | 2011-01-01 tot 2014-08-01 |
-      | adresseerbaarObjectIdentificatie | 0800000000000001          |
-      En heeft de bewoning een bewoner met de volgende gegevens
-      | burgerservicenummer |
-      | 000000024           |
-      En heeft de bewoner de volgende 'naam' gegevens
-      | naam                                 | waarde    |
-      | voornamen                            | Robin Sam |
-      | adellijkeTitelPredicaat.code         | B         |
-      | adellijkeTitelPredicaat.omschrijving | baron     |
-      | adellijkeTitelPredicaat.soort        | titel     |
-      | voorvoegsel                          | van den   |
-      | geslachtsnaam                        | Aedel     |
-      En heeft de bewoner de volgende 'geslacht' gegevens
-      | naam         | waarde |
-      | code         | M      |
-      | omschrijving | man    |
-      En heeft de response een bewoning met de volgende gegevens
-      | naam                             | waarde                    |
-      | periode                          | 2010-09-01 tot 2011-01-01 |
-      | adresseerbaarObjectIdentificatie | 0800000000000001          |
-      En heeft de bewoning een mogelijke bewoner met de volgende gegevens
-      | burgerservicenummer | naam.voornamen | naam.voorvoegsel | naam.geslachtsnaam | naam.adellijkeTitelPredicaat.code | naam.adellijkeTitelPredicaat.omschrijving | naam.adellijkeTitelPredicaat.soort | geslacht.code | geslacht.omschrijving |
-      | 000000024           | Robin Sam      | van den          | Aedel              | B                                 | baron                                     | titel                              | M             | man                   |
+    Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
+    | voornamen (02.10) | adellijke titel of predicaat (02.20) | voorvoegsel (02.30) | geslachtsnaam (02.40) | geslachtsaanduiding (04.10) |
+    | Robin Sam         | B                                    | van den             | Aedel                 | M                           |
+    En de persoon is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100000                           |
+    Als bewoningen wordt gezocht met de volgende parameters
+    | naam                             | waarde             |
+    | type                             | BewoningMetPeriode |
+    | datumVan                         | 2010-09-01         |
+    | datumTot                         | 2014-08-01         |
```

Comment:
ik zou de datumTot wijzigen tot bijv. 2011-01-01, zodat de gevraagde periode alleen in de onzekerheidsperiode valt. Hierdoor bevat de response alleen mogelijke bewoners en wordt daadwerkelijk alleen gevalideerd dat voor een mogelijke bewoner ook naam en geslachtsaanduiding velden wordt geleverd